app/assets/stylesheets/refills/_header.scss

```diff
@@ -0,0 +1,87 @@
+.header-2 {
+  $base-spacing: 1.5em !default;
+  $base-accent-color: $bookshelf-red-2 !default;
+  $medium-screen: em(640) !default;
+  $large-screen: em(860) !default;
+
+  ul {
+    margin: 0;
+    padding: 0;
+    line-height: 1.5em;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  $header-background: $base-accent-color;
+  $header-color: white;
+  $header-link-color: transparentize($header-color, 0.2);
+  $header-disclaimer-color: transparentize($header-color, 0.6);
+
+  background: $header-background;
```

Comment:
Expected item on line 26 to appear before line 7. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets